reference/TEMPLATE.md

```diff
@@ -37,11 +37,11 @@
 - Caveat2: {Description}
```

Comment:
```suggestion
- Caveats: {Description}
```

Reply:
There is caveat 1 and caveat 2, so I will leave it as it is
```
- Caveat1: {Description}
- Caveat2: {Description}
```